src/pages/UpSertUser/hooks/useInsertUser.ts

```diff
@@ -0,0 +1,61 @@
+import { gql } from "graphql-request";
+import { useMutation } from "react-query";
+import {
+    InsertUserMutationMutation,
+    InsertUserMutationMutationVariables,
+} from "src/generated/graphql";
+
+import useQueryClient from "src/hooks/useQueryClient";
+
+const useInsertUser = (
+    backToListPage: () => void,
+    showError: (message: string, type: "error" | "success") => void
+) => {
+    const queryClient = useQueryClient();
+    const result = useMutation<
+        InsertUserMutationMutation,
+        { response: any },
```

Comment:
I see "any", I comment

Reply:
![image](https://user-images.githubusercontent.com/58205909/131880023-e9d8a61b-b1ef-4eae-b8b9-97b2c8143077.png)
This is type for error response of Hasura I found in Hasura.io